plugin/trino-hive/src/main/java/io/trino/plugin/hive/security/AccessControlMetadata.java

```diff
@@ -34,6 +35,11 @@ default boolean isUsingSystemSecurity()
         return false;
     }
 
+    default boolean supportsRoles()
```

Comment:
I don't like it. Maybe we should have `AccessControlMetadata` to be bound as `Optional`?